### PR TITLE
Feature: app folder translation

### PR DESCRIPTION
**Pull request by MercierCorentin**
In this PR @CdRom1 and I translated the `app` folder.
Part of #242 resolution.

### CODE REVIEW

**Comment by r0one** [comment]
@MercierCorentin you did almost everything and that is a huuuge work gg. You did some typos but they don't prevent from understanding so I won't fix them. Let me just push a few fixes and I'll approve this PR.

**Comment by r0one** [comment]
@MercierCorentin you only had forgotten **one line** amongst *thousands* 👏💪😗😱

**Review by r0one — DISMISSED** [review]

File: app/Admin/Controllers/Management/AccessController.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Permet de gérer les accès.
+ * Manages accesses.
```
Comment (NastuzziSamy) [inline app/Admin/Controllers/Management/AccessController.php]: Same for everyone, not 3rd person

**Review by NastuzziSamy — CHANGES_REQUESTED** [review]
Huge PR, need to fix the 3rd person before

**Pull request merged**